<launch>
  <node pkg="ccny_karto" type="scan_matcher_node" name="scan_matcher_node" output="screen">
    <param name="history_length" value="15"/>
    <param name="history_skip"   value="3"/>
    <param name="publish_tf"     value="true"/>
    <param name="search_sizes"     type="string" value="0.40 0.10"/>
    <param name="resolutions"      type="string" value="0.04 0.01"/>
    <param name="smear_deviations" type="string" value="0.04 0.01"/>

  </node>
  
  <node pkg="hokuyo_node" type="hokuyo_node" name="hokuyo_node" output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz"/>

  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" 
    args="0 0 0.0 0 0 0 /base_link /laser 40" />

  <node pkg="tf" type="static_transform_publisher" name="odom_to_base_link" 
    args="0 0 0 0 0 0 /odom /base_link 40" />

</launch>
